moveit_core/robot_model/include/moveit/robot_model/robot_model.h

```diff
@@ -61,6 +61,15 @@ namespace core
 {
 MOVEIT_CLASS_FORWARD(RobotModel);  // Defines RobotModelPtr, ConstPtr, WeakPtr... etc
 
+static inline void checkInterpolationParamBounds(const char LOGNAME[], const double& t)
```

Comment:
Don't pass simple data types as double by reference:
https://stackoverflow.com/questions/7561690/why-do-we-not-pass-pod-by-reference-in-functions